tests/gradients/parameter_shift/test_parameter_shift.py

```diff
@@ -3405,12 +3404,12 @@ def test_torch(self, dev_name, tol, broadcast):
 
         dev = qml.device(dev_name, wires=2)
 
-        if broadcast:
-            with pytest.raises(
-                NotImplementedError, match="Broadcasting with multiple measurements"
-            ):
-                res = self.cost_fn(weights, coeffs1, coeffs2, dev, broadcast)
-            return
+        # if broadcast:  # breaks xfail_strict
+        #     with pytest.raises(
+        #         NotImplementedError, match="Broadcasting with multiple measurements"
+        #     ):
+        #         res = self.cost_fn(weights, coeffs1, coeffs2, dev, broadcast)
+        #     return
```

Comment:
```
if broadcast:
    ....
    return 
pytest.xfail("some message")
```

only mark the non-broadcast case `xfail`.

Reply:
sure. it's kinda clumsy because the parametrization is for the whole class, and using `pytest.xfail` at some point within a test makes you miss the value-add of `xfail_strict`